### PR TITLE
Rosettacode: Create HTML tables

### PR DESCRIPTION
**Pull request by MehdiAit**
Signed-off-by: Mehdi <overpex@gmail.com>

### CODE REVIEW

File: examples/rosettacode/html_table.nit [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This file is part of NIT ( http://www.nitlanguage.org )
+# This program is public domain
+#
+# Task: Create HTML tables
+# SEE: <http://rosettacode.org/wiki/Create_an_HTML_table>
+
+fun creat_table (s: String, ligne:String): String do
```
Comment (privat) [inline examples/rosettacode/html_table.nit]: e?

File: examples/rosettacode/html_table.nit [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	for x in [0..i.length[ do
+		s = s + "<th>{i[x]}</th>"
+	end
+	return s
+end
+
+
+var file = new FileWriter.open("table.html")
+file.write(create_table("A;B;C;D", "[1,2,3,4]; [5,2,3,8]"))
+file.close
```
Comment (privat) [inline examples/rosettacode/html_table.nit]: shorter:

~~~nit
create_table("A;B;C;D", "[1,2,3,4]; [5,2,3,8]").write_to_file("table.html")
~~~

File: examples/rosettacode/html_table.nit [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# Task: Create HTML tables
+# SEE: <http://rosettacode.org/wiki/Create_an_HTML_table>
+
+fun create_table (s: String, ligne:String): String do
+	var table = ""
+	var colum =  get_colum(s.split(";"))
+	table = "<table style=\"text-align:center; border: 1px solid\">{colum}"
+	var values = get_ligne(ligne.split(";"))
+	table = table + "\n" + values + "\n</table>"
+	print table
```
Comment (privat) [inline examples/rosettacode/html_table.nit]: wat? `print`?

File: examples/rosettacode/html_table.nit [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	return table
+end
+
+fun get_ligne(s: Array[String]): String do
+	var c = ""
+	for x in [0..s.length[ do
+		c = c + "<tr>"
+		s[x] = s[x].replace('[', "")
+		s[x] = s[x].replace(']', "")
+		s[x] = s[x].replace(' ', "")
```
Comment (privat) [inline examples/rosettacode/html_table.nit]: not that great but no better idea

File: examples/rosettacode/html_table.nit [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+		
+		var a = s[x].split(",")
+		for y in [0..a.length[ do
+			c = c + "<td>{a[y]}</td>"
+		end
```
Comment (privat) [inline examples/rosettacode/html_table.nit]:
~~~nit
for y in a do c += "<td>{y}</td>"
~~~

File: examples/rosettacode/html_table.nit (hunk 4 above)
Comment (MehdiAit) [inline examples/rosettacode/html_table.nit]: I know! but it the shorter and easier way that I've found !

File: examples/rosettacode/html_table.nit [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+		c = c + "</tr>"
+	end
+	return c
+end
+
+fun get_colum(i: Array[String]): String do
+	var s = ""
+	for x in [0..i.length[ do
+		s = s + "<th>{i[x]}</th>"
```
Comment (privat) [inline examples/rosettacode/html_table.nit]: use `for x in i` but also choose better variable names. please.

**Comment by ventilooo** [comment]
+1

**Comment by MehdiAit** [comment]
ping @privat

**Comment by privat** [comment]
+1

File: examples/rosettacode/html_table.nit [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# This file is part of NIT ( http://www.nitlanguage.org )
+# This program is public domain
+#
+# Task: Create HTML tables
+# SEE: <http://rosettacode.org/wiki/Create_an_HTML_table>
+
+fun create_table (s: String, ligne:String): String do
```
Comment (Tagachi) [inline examples/rosettacode/html_table.nit]: Either space before type or not.
But used the same convention.

File: examples/rosettacode/html_table.nit [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+	var table = ""
+	var colum =  get_colum(s.split(";"))
```
Comment (Tagachi) [inline examples/rosettacode/html_table.nit]: colum or column?
Too much blank space after ```=```.

File: examples/rosettacode/html_table.nit (hunk 7 above)
Comment (Tagachi) [inline examples/rosettacode/html_table.nit]: Same comment for blank space before ```(```.

File: examples/rosettacode/html_table.nit [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+	var values = get_ligne(ligne.split(";"))
+
+	table = "<table style=\"text-align:center; border: 1px solid\">{columns}"
+	table = table + "\n" + values + "\n</table>"
+	return table
+end
+
+fun get_ligne(other: Array[String]): String do
+	var values = ""
+	for x in [0..other.length[ do
```
Comment (Tagachi) [inline examples/rosettacode/html_table.nit]: Replace ```[0..other.length[``` by ```other.length.times```

**Comment by privat** [comment]
closed by 305f372e18dee0b8ddf0d3df2c28c1f2c127977a

**Pull request closed without merging**